docs/guides/file-system-based-automated-bundle-generation.md

```diff
@@ -0,0 +1,133 @@
+# File-System-Based Automated Bundle Generation
+
+To use the automated bundle generation feature introduced in React on Rails v14.0.0, Please upgrade to use [Shakapacker v6.5.0](https://github.com/shakacode/shakapacker/tree/v6.5.0) at least. If you are currently using webpacker, please follow the migration steps available [here](https://github.com/shakacode/shakapacker/blob/master/docs/v6_upgrade.md).
+
+## Configuration
+
+### Enable nested_entries for Shakapacker
+To use the automated bundle generation feature, in the `webpacker.yml` file, set 
+
+```yml
+default:
+  ...
+  nested_entries: true
+```
+
+For more details, see [Configuration and Code ](https://github.com/shakacode/shakapacker#configuration-and-code) section in [shakapacker](https://github.com/shakacode/shakapacker/).
+
+### Configure Components Directory
+`components_directory` is the directory used to automatically detect and register components for usage on the rails view.
```

Comment:
Rails should be capitalized. 

I don't like the phrasing because below you show multiple `ror_components` directories, so not really "the" directory. How about

> `components_directory` is the name for directories containing components which can be automatically registered and used in Rails views

?